File: src/main/id2h/yatm/item/ItemPurifiedUraniumCrystal.scala
package id2h.yatm.item

import java.util.EnumSet

import appeng.api.recipes.ResolverResult
import appeng.core.features.AEFeature
import appeng.items.AEBaseItem

class ItemPurifiedUraniumCrystal extends AEBaseItem() {
	this.setFeature(EnumSet.of(AEFeature.Core));
	this.setTextureName("yatm:ItemMaterial.PurifiedUraniumCrystal")
	this.setUnlocalizedName("yatm.ItemPurifiedUraniumCrystal")
}

File: src/main/id2h/yatm/common/item/ItemPurifiedUraniumCrystal.java
package id2h.yatm.common.item;

import java.util.EnumSet;

import appeng.core.features.AEFeature;
import appeng.items.AEBaseItem;

public class ItemPurifiedUraniumCrystal extends AEBaseItem
{
	public ItemPurifiedUraniumCrystal()
	{
		super();
		this.setFeature(EnumSet.of(AEFeature.Core));
		this.setTextureName("yatm:ItemMaterial.PurifiedUraniumCrystal");
		this.setUnlocalizedName("yatm.ItemPurifiedUraniumCrystal");
	}
}
